components/org.wso2.carbon.appmgt.mdm.osgiconnector/src/main/java/org/wso2/carbon/appmgt/mdm/osgiconnector/mdmmgt/beans/android/AppStoreApplication.java

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
There are some common properties in these 3 classes. Better to introduce the super class.
beans/android/AppStoreApplication
beans/android/EnterpriseApplication
beans/android/WebApplication